Common/column.cpp

```diff
@@ -1014,8 +1014,8 @@ string Column::_getScaleValue(int row)
 {
 	double v = AsDoubles[row];
 
-	if (v > DBL_MAX)					return string({ (char)0xE2, (char)0x88, (char)0x9E, 0 });
-	else if (v < -DBL_MAX)				return string({ (char)0x2D, (char)0xE2, (char)0x88, (char)0x9E, 0 });
+	if (v > std::numeric_limits<double>::max())					return string({ (char)0xE2, (char)0x88, (char)0x9E, 0 });
+	else if (v < std::numeric_limits<double>::min())			return string({ (char)0x2D, (char)0xE2, (char)0x88, (char)0x9E, 0 });
```

Comment:
```suggestion
	else if (v < std::numeric_limits<double>::lowest())			return string({ (char)0x2D, (char)0xE2, (char)0x88, (char)0x9E, 0 });
```
see https://en.cppreference.com/w/cpp/types/numeric_limits. `lowest()` returns the largest negative value (or zero) whereas `min()` returns the smallest positive value. Although `std::numeric_limits<int>::min() == std::numeric_limits<int>::lowest()`, it could make sense to use `std::numeric_limits<int>::lowest()` everywhere since people are guaranteed to use `std::numeric_limits<double>::min()` by mistake if they see `std::numeric_limits<int>::min()` being used.

Reply:
You are 100% right! 👍🏼